packages/core/src/rpc/index.ts

```diff
@@ -135,6 +205,11 @@ export const createRequestQueue = ({
     ) => {
       const stopClockLag = startClock();
 
+      return requestQueue.add({ request: params, stopClockLag });
+    },
+    subscribe: (params: SubscribeParameters) => {
+      const stopClockLag = startClock();
+
       return requestQueue.add({ request: params, stopClockLag });
```

Comment:
Thinking about `subscribe()` not adding to the queue. I think it's safe to assume that we aren't going to get a 429 with the requests sent to it, because there should just be one per chain.

Reply:
Yeah that makes sense.